tests/test_ml.py

```diff
@@ -209,17 +244,35 @@
     'tags': TAGS,
     'tfliteModel': TFLITE_FORMAT_JSON
 }
-FULL_MODEL_PUBLISHED = ml.Model.from_dict(FULL_MODEL_PUBLISHED_JSON)
-OPERATION_DONE_FULL_MODEL_PUBLISHED_JSON = {
+FULL_TFLITE_MODEL_PUBLISHED = ml.Model.from_dict(FULL_TFLITE_MODEL_PUBLISHED_JSON)
+OPERATION_DONE_FULL_TFLITE_MODEL_PUBLISHED_JSON = {
+    'name': OPERATION_NAME_1,
+    'done': True,
+    'response': FULL_TFLITE_MODEL_PUBLISHED_JSON
+}
+FULL_COREML_MODEL_PUBLISHED_JSON = {
+    'name': MODEL_NAME_1,
+    'displayName': DISPLAY_NAME_1,
+    'createTime': CREATE_TIME,
+    'updateTime': UPDATE_TIME,
+    'state': MODEL_STATE_PUBLISHED_JSON,
+    'etag': ETAG,
+    'modelHash': MODEL_HASH,
+    'tags': TAGS,
+    'coremlModel': COREML_FORMAT_JSON
+}
+FULL_COREML_MODEL_PUBLISHED = ml.Model.from_dict(FULL_COREML_MODEL_PUBLISHED_JSON)
+OPERATION_DONE_FULL_COREML_MODEL_PUBLISHED_JSON = {
     'name': OPERATION_NAME_1,
     'done': True,
-    'response': FULL_MODEL_PUBLISHED_JSON
+    'response': FULL_COREML_MODEL_PUBLISHED_JSON
 }
 
 EMPTY_RESPONSE = json.dumps({})
 OPERATION_NOT_DONE_RESPONSE = json.dumps(OPERATION_NOT_DONE_JSON_1)
 OPERATION_DONE_RESPONSE = json.dumps(OPERATION_DONE_MODEL_JSON_1)
-OPERATION_DONE_PUBLISHED_RESPONSE = json.dumps(OPERATION_DONE_FULL_MODEL_PUBLISHED_JSON)
+OP_DONE_TFLITE_PUBLISHED_RESPONSE = json.dumps(OPERATION_DONE_FULL_TFLITE_MODEL_PUBLISHED_JSON)
+OP_DONE_COREML_PUBLISHED_RESPONSE = json.dumps(OPERATION_DONE_FULL_COREML_MODEL_PUBLISHED_JSON)
```

Comment:
You can remove the `_JSON` constant by defining this as an inlined API call:

```py
OP_DONE_COREML_PUBLISHED_RESPONSE = json.dumps({
 # ...
})
```

Reply:
done